app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float, "Distance")) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: (int, float, Distance)) -> Distance:
```

Reply:
use `from __future__ import annotations`

Reply:
Ye, sorry, I first wrote this code and only then saw another lesson.